src/Footer.css

```diff
@@ -0,0 +1,41 @@
+#footer {
```

Comment:
Comment categorizing this as scaffolding.

Reply:
Done.